tests/issues.targets

```diff
@@ -81,6 +81,9 @@
         <ExcludeList Include="$(XunitTestBinBase)/JIT/Methodical/tailcall_v4/hijacking/*">
             <Issue>Unix does not support tailcall helper</Issue>
         </ExcludeList>
+        <ExcludeList Include="$(XunitTestBinBase)/JIT/Directed/tailcall/more_tailcalls/*">
```

Comment:
I think we have an all unix section that is probably worth just adding this once

Reply:
There is one but I have just added it below where other tests were disabled due to lack of tailcall helper. What do you prefer? (note that this needs to be disabled for arm64 windows too, although I realize there is no testing for it)

Reply:
Ah I see the reasoning. Keep it where it is, soon we can wack all that cruft.